tests/Unit/Infrastructure/DefinitionExtractorTest.php

```diff
@@ -49,22 +49,22 @@ public function testResolveConstructor(): void
     public function testResolveCarConstructor(): void
     {
         $resolver = DefinitionExtractor::getInstance();
-        $dependencyResolver = new SimpleDependencyResolver();
+        $container = new SimpleContainer();
         /** @var DefinitionInterface[] $dependencies */
         $dependencies = $resolver->fromClassName(Car::class);
 
         $this->assertCount(2, $dependencies);
         $this->assertInstanceOf(ClassDefinition::class, $dependencies['engine']);
         $this->assertInstanceOf(ParameterDefinition::class, $dependencies['moreEngines']);
 
-        $this->expectException(NotFoundException::class);
-        $dependencies['engine']->resolve($dependencyResolver);
+        $this->expectException('Yiisoft\Test\Support\Container\Exception\NotFoundException');
```

Comment:
```suggestion
        $this->expectException(NotFoundException::class);
```

Reply:
You are wrong, we have two different  `NotFoundException` classes in this test.

Reply:
Use full name `\Yiisoft\Test\Support\Container\Exception\NotFoundException::class`. Is better that string.